stk/calculators/optimization/optimizers.py

```diff
@@ -700,6 +703,11 @@ class GFNXTB(Optimizer):
     gfnxtb_path : :class:`str`
         The path to the GFN-xTB executable.
 
+    output_dir : :class:`str`
+        The name of the directory into which files generated during
+        the optimization are written, if ``None`` then
+        :func:`uuid.uuid4` is used.
+
     num_cores : :class:`str`
         The number of cores for GFN-xTB to use.
 
```

Comment:
in the GFN example, there should be a note that the lack of bonding information means the structure should be pre-optimized somehow (ETKDG for example)

Reply:
This is an excellent point.